marlowe-playground-client/src/SimulationPage/View.purs

```diff
@@ -40,15 +44,28 @@ render ::
 render state =
   div [ classes [ fullHeight, scroll, ClassName "simulation-panel" ] ]
     [ section [ class_ (ClassName "code-panel") ]
-        [ div [ classes (codeEditor $ state ^. _showBottomPanel) ]
+        [ div [ classes [ codeEditor ] ]
             [ marloweEditor state ]
         , sidebar state
         ]
-    , bottomPanel state
+    , renderSubmodule _bottomPanelState BottomPanelAction (BottomPanel.render panelTitles wrapBottomPanelContents) state
     ]
   where
+  panelTitles =
+    [ { title: "Current State", view: CurrentStateView, classes: [] }
+    ]
+
+  currentStateClasses = if hasRuntimeWarnings || hasRuntimeError then [ ClassName "error-tab" ] else []
+
+  -- QUESTION: what are runtimeWarnings and runtimeError? how can I reach that state?
```

Comment:
best to ask @palas for some examples, these days it's quite hard to achieve an error (impossible?) or a warning (that's a good thing!)